pkg/ctl/create/cluster.go

```diff
@@ -354,6 +355,23 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 				return fmt.Errorf("failed to create addons")
 			}
 		}
+		// After we have the cluster config and all the nodes are done, we install Karpenter if necessary.
+		if cfg.Karpenter != nil {
+			karpenterTaskTree := stackManager.NewTasksToInstallKarpenter()
+			logger.Info(karpenterTaskTree.Describe())
+			if errs := karpenterTaskTree.DoAllSync(); len(errs) > 0 {
+				logger.Warning("%d error(s) occurred and cluster hasn't been created properly, you may wish to check CloudFormation console", len(errs))
```

Comment:
This message needs to be updated to be more Karpenter specific because at this point, the cluster and nodegroups will have been created successfully.